src/index.html

```diff
@@ -15,8 +15,75 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header is-active1">
+      <a
+        href="#"
+        class="header-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <a
+          href="#section-1"
+          class="nav__link nav__link1 is-active"
```

Comment:
Also you don't need number of link. you can use `nav__link` and `is-active` for all requiremets which be in this task.